dev-libs/libconfig/libconfig-1.5.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A simple C/C++ configuration file library"
+DESCRIPTION="Libconfig is a simple library for processing structured \
+configuration files. Libconfig is very compact, a fraction of the size \
+of the expat XML parser library. This makes it well-suited for \
+memory constrained systems like handheld devices."
```

Comment:
Could you please change the line wrapping a little in DESCRIPTION? The ideal wrapping is:
```
DESCRIPTION="Libconfig is a simple library for processing structured \
configuration files. Libconfig is very compact, a fraction of the size of the \
expat XML parser library. This makes it well-suited for memory constrained \
systems like handheld devices."
```